examples/md-flexible/src/Simulation.cpp

```diff
@@ -21,346 +22,235 @@ extern template bool autopas::AutoPas<ParticleType>::iteratePairwise(autopas::LJ
+    : _configuration(configuration),
+      _domainDecomposition(domainDecomposition),
+      _createVtkFiles(not configuration.vtkFileName.value.empty()),
+      _vtkWriter(std::make_shared<ParallelVtkWriter>(_configuration.vtkFileName.value,
+                                                     _configuration.vtkOutputFolder.value,
+                                                     std::to_string(_configuration.iterations.value).size())) {
+  _timers.total.start();
+  _timers.initialization.start();
+
+  if (_configuration.logFileName.value.empty()) {
+    _outputStream = &std::cout;
+  } else {
+    _logFile = std::make_shared<std::ofstream>();
+    _logFile->open(_configuration.logFileName.value);
+    _outputStream = &(*_logFile);
+  }
+
+  _autoPasContainer = std::make_shared<autopas::AutoPas<ParticleType>>(*_outputStream);
+  _autoPasContainer->setAllowedCellSizeFactors(*_configuration.cellSizeFactors.value);
+  _autoPasContainer->setAllowedContainers(_configuration.containerOptions.value);
+  _autoPasContainer->setAllowedDataLayouts(_configuration.dataLayoutOptions.value);
+  _autoPasContainer->setAllowedNewton3Options(_configuration.newton3Options.value);
+  _autoPasContainer->setAllowedTraversals(_configuration.traversalOptions.value);
+  _autoPasContainer->setAllowedLoadEstimators(_configuration.loadEstimatorOptions.value);
+  _autoPasContainer->setBoxMin(_domainDecomposition.getLocalBoxMin());
+  _autoPasContainer->setBoxMax(_domainDecomposition.getLocalBoxMax());
+  _autoPasContainer->setCutoff(_configuration.cutoff.value);
+  _autoPasContainer->setRelativeOptimumRange(_configuration.relativeOptimumRange.value);
+  _autoPasContainer->setMaxTuningPhasesWithoutTest(_configuration.maxTuningPhasesWithoutTest.value);
+  _autoPasContainer->setRelativeBlacklistRange(_configuration.relativeBlacklistRange.value);
+  _autoPasContainer->setEvidenceFirstPrediction(_configuration.evidenceFirstPrediction.value);
+  _autoPasContainer->setExtrapolationMethodOption(_configuration.extrapolationMethodOption.value);
+  _autoPasContainer->setNumSamples(_configuration.tuningSamples.value);
+  _autoPasContainer->setMaxEvidence(_configuration.tuningMaxEvidence.value);
+  _autoPasContainer->setSelectorStrategy(_configuration.selectorStrategy.value);
+  _autoPasContainer->setTuningInterval(_configuration.tuningInterval.value);
+  _autoPasContainer->setTuningStrategyOption(_configuration.tuningStrategyOption.value);
+  _autoPasContainer->setMPIStrategy(_configuration.mpiStrategyOption.value);
+  _autoPasContainer->setVerletClusterSize(_configuration.verletClusterSize.value);
+  _autoPasContainer->setVerletRebuildFrequency(_configuration.verletRebuildFrequency.value);
+  _autoPasContainer->setVerletSkin(_configuration.verletSkinRadius.value);
+  _autoPasContainer->setAcquisitionFunction(_configuration.acquisitionFunctionOption.value);
+  _autoPasContainer->init();
+
+  autopas::Logger::get()->set_level(_configuration.logLevel.value);
```

Comment:
```suggestion
  autopas::Logger::get()->set_level(_configuration.logLevel.value);
  _autoPasContainer->init();
```

the `init()` function has a logger call that would be unaffected if the level is set afterward.